_sass/blocks/_chart-list.scss

```diff
@@ -28,7 +37,12 @@
   .chart-item {
     @include span-columns(4);
 
+    -ms-flex-direction: column;
+    -webkit-flex-direction: column;
```

Comment:
It looks like flexbox unprefixed is currently [_almost completely_](http://caniuse.com/#feat=flexbox) covered, but adding prefixes will be nice for the few browsers that it only partially supports unprefixed